tdrs-frontend/nginx/buildpack.nginx.conf

```diff
@@ -28,9 +28,25 @@ log_format compression '$remote_addr - $remote_user [$time_local] '
         root public;
         listen {{port}};
         access_log /home/vcap/app/nginx_access.log compression;
+
         include locations.conf;
-        add_header Content-Security-Policy "default-src 'none'; script-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self'; connect-src 'self' *.acf.hhs.gov *.cloud.gov; manifest-src 'self'; object-src 'none'; frame-ancestors 'none'; form-action 'none';";
+
+        set $CONNECT_SRC {{env "CONNECT_SRC"}};
+        set $ALLOWED_ORIGIN {{env "ALLOWED_ORIGIN"}};
+
+        set $CSP "default-src 'none';";
+        set $CSP "${CSP}script-src 'self';";
+        set $CSP "${CSP}style-src 'self' 'unsafe-inline';";
+        set $CSP "${CSP}img-src 'self' data:;";
+        set $CSP "${CSP}font-src 'self';";
+        set $CSP "${CSP}connect-src 'self' ${CONNECT_SRC};";
+        set $CSP "${CSP}manifest-src 'self';";
+        set $CSP "${CSP}object-src 'none';";
+        set $CSP "${CSP}frame-ancestors 'none';";
+        set $CSP "${CSP}form-action 'none';";
+
+        add_header Content-Security-Policy "${CSP}";
         add_header Cache-Control "no-store, no-cache, must-revalidate, max-age=0";
-        add_header Access-Control-Allow-Origin "https://tanfdata.acf.hhs.gov";
+        add_header Access-Control-Allow-Origin "${ALLOWED_ORIGIN}";
```

Comment:
I took all of this from an older version of the app. `Access-Control-Allow-Origin` is important for logging in I believe.

Reply:
`django-cors-headers` has a setting for allowed origins
`CORS_ALLOWED_ORIGINS`
https://pypi.org/project/django-cors-headers/

do we have to use nginx to set the `Access-Control-Allow-Origin` header?